jboss-standalone/roles/jboss-standalone/handlers/main.yml

```diff
@@ -1,10 +1,12 @@
 ---
-- name: restart jboss
-  service:
+- name: Restart jboss
+  ansible.builtin.service:
     name: jboss
     state: restarted
+  listen: restart_jboss
 
-- name: restart iptables
-  service:
+- name: Restart iptables
+  ansible.builtin.service:
     name: iptables
     state: restarted
+  listen: restart_iptables
```

Comment:
Why use snake_case for handlers? They aren't Python identifiers. It'd be confusing.